tests/python/unittest/test_random.py

```diff
@@ -871,14 +871,15 @@ def test_randint_generator():
             buckets = np.array(buckets, dtype=dtype).tolist()
             probs = [(buckets[i][1] - buckets[i][0]) / float(scale) for i in range(5)]
             generator_mx = lambda x: mx.nd.random.randint(low, high, shape=x, ctx=ctx, dtype=dtype).asnumpy()
-            verify_generator(generator=generator_mx, buckets=buckets, probs=probs)
+            verify_generator(generator=generator_mx, buckets=buckets, probs=probs, alpha=0.01)
```

Comment:
suggest to try if we can increase alpha, I also feel nrepeat=5 is a little too small that might cause flasky.

Reply:
updated it